.circleci/config.yml

```diff
@@ -43,7 +44,9 @@ jobs:
             pip install --upgrade pip
             pip install Cython
             pip install -e .
-            python .circleci/test_script.py
+            TESTFILES=$(circleci tests glob examples/*.py | circleci tests split --split-by=timings)
```

Comment:
Is this the thing allowing to do a matrix in circle ci?

Reply:
This is what will split the python scripts into 4 batches, based on previous run times..

Reply:
oh nice that it's based on previous run times ! Amazing